src/cache-updater/index.ts

```diff
@@ -123,10 +123,18 @@ function getRepos(): RepoInfo[] {
   }
 }
 
-const repos = getRepos();
+async function main() {
+  const repos = getRepos();
 
-repos.forEach((repo) =>
-  handler(repo).catch((error) => {
-    console.error('An error occurred!', error);
-  })
-);
+  await Promise.all(
```

Comment:
Added this because the job seemed to be running for a long time after the uploads were complete.